README.md

```diff
@@ -172,6 +172,10 @@ else
 fi
 ```
 
+You can completely omit the `docker/build` script if you don't need any extra
```

Comment:
Ah, ok. I suggested this in the previous commit (where it should be included I think).